beacon_node/beacon_chain/src/light_client_server_cache.rs

```diff
@@ -276,10 +276,24 @@ impl<T: BeaconChainTypes> LightClientServerCache<T> {
     ) -> Result<Vec<LightClientUpdate<T::EthSpec>>, BeaconChainError> {
         let column = DBColumn::LightClientUpdate;
         let mut light_client_updates = vec![];
-        for res in store
-            .hot_db
-            .iter_column_from::<Vec<u8>>(column, &start_period.to_le_bytes())
-        {
+
+        let results = store.hot_db.iter_column_from::<Vec<u8>>(
+            column,
+            &start_period.to_le_bytes(),
+            move |sync_committee_bytes, _| {
+                let Ok(sync_committee_period) = u64::from_ssz_bytes(sync_committee_bytes) else {
```

Comment:
Is it safe to silence this error?

Reply:
It's probably safe to silence this error, but I'm now logging the error just in case
[8ed2ff4](https://github.com/sigp/lighthouse/pull/4718/commits/8ed2ff4763b2afde395eebb830c11a8101b471ba)